### PR TITLE
feat: revamping the Navbar #1195

### PR DESCRIPTION
**Pull request by pranay911**
Changed the color of the navigation item and Added a navigation button according to design.

### 📸 Screenshots 
 

image

### CODE REVIEW

**Comment by pranay911** [comment]
Hi @tamalCodes I've considered all the approaches you mentioned. In the provided image, there's only one item on the right side. However, on your website, two buttons – "Contribute to Milan" and "Sign Up" – occupy some space. Due to this, the navigation items are not centered as expected.

File: src/components/Button/GlobalButton/Button.module.css [hunk 1]
```diff
@@ -1,12 +1,12 @@
 .btn {
-  padding: 8px 15px;
+  padding: 8px 25px;
   text-align: center;
   cursor: pointer;
-  border-radius: 8px;
+  border-radius: 40px;
```
Comment (tamalCodes) [inline src/components/Button/GlobalButton/Button.module.css:5]: The radius might be different at different places so make another variant that also defines the border radius of the button for example you can make a class called `rounded-button` which has a different border radius and the default button can have the radius that we previously had.

We can use this class in places where we need it.

File: src/components/Navbar/Navbar.jsx [hunk 2]
```diff
@@ -127,14 +128,20 @@ const Navbar = () => {
...
+                      variant="Primary"
                       to="/auth/login"
                       fontweight="bold"
                     >
-                      Sign in
+                      <span>
+                        Contribute to Milan
+                        <IoChevronForwardSharp style={{ marginLeft: "5px" }} />
+                      </span>
                     </Button>
```
Comment (tamalCodes) [inline src/components/Navbar/Navbar.jsx:139]: As discussed, for now let us hide this

**Review by tamalCodes — CHANGES_REQUESTED** [review]

File: src/components/Button/GlobalButton/Button.module.css (hunk 1 above)
Comment (pranay911) [reply src/components/Button/GlobalButton/Button.module.css:5]: U mean hide contribute to milan button?

Comment (pranay911) [reply src/components/Button/GlobalButton/Button.module.css:5]: So basically I create a another button with different border-radius...

**Comment by pranay911** [comment]
Hey @tamalCodes, good evening! Following our discussion, I've made the following changes:
1. Added a new class for rounded buttons.
2. Corrected the border-radius.

**Comment by pranay911** [comment]
Hey @tamalCodes, any updates on PR?

File: src/components/Navbar/Navbar.jsx (hunk 2 above)
Comment (tamalCodes) [reply src/components/Navbar/Navbar.jsx:139]: @pranay911 , you said that you would hide it - but it is still visible.
image

Also when hovering over the button everything disappears, add a border for the hovering instance. Also please make the navbar texts be of a lesser font-weight, 400 would be great.
image

**Comment by pranay911** [comment]
**Now its looks Pretty**

image

**Review by tamalCodes — APPROVED** [review]
# Amazing work 🎉
**The changes look good to me and will be merged soon.**

Do follow [Tamal](https://github.com/tamalCodes) for more Opensource fun projects and don't forget to drop a star so that you get updated about our latest [releases](https://github.com/MilanCommunity/Milan/releases) (we will tag you and mention your work) and also a [shoutout](https://www.linkedin.com/feed/update/urn:li:activity:7073574804519354368/) on social media (LinkedIn and Twitter) !

**PLEASE DROP US A STAR, MEANS ALOT.**

Happy Opensource 🚀.

**Pull request merged**